gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat Aug 26 14:22:02 EDT 2017
+#Thu Oct 12 17:55:37 CDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-all.zip
```

Comment:
Since this is on 4.0, i might just update this to 4.2 as it should be more stable (Will test this to be sure it still properly builds)

Reply:
4.0-all to 4.2.1-all to be precise (Unless 4.3 releases within the next day xD)

Reply:
Why switch Gradle from bin to all?

Reply:
Tbh, idk myself. Seems IntelliJ IDEA always suggest it's better to use -All, but since I don't think it's really needed I'll probably rebert that